packages/fxa-auth-server/.eslintrc.json

```diff
@@ -5,7 +5,13 @@
   "extends": ["plugin:fxa/server"],
   "plugins": ["fxa"],
   "parserOptions": {
-    "ecmaVersion": "2020",
+    "ecmaVersion": 2020,
     "sourceType": "module"
-  }
+  },
+  "ignorePatterns": [
+    "dist",
+    "fxa-*",
+    "vendor"
```

Comment:
Added "vendor" here since it was linting some vendor/ejs.js file, per https://github.com/mozilla/fxa/blob/397c5906816518658b821278513ce371bdad322a/packages/fxa-auth-server/scripts/install-ejs.sh.